app/ante/eth.go

```diff
@@ -464,12 +429,13 @@ func (vbd EthValidateBasicDecorator) AnteHandle(ctx sdk.Context, tx sdk.Tx, simu
 			return ctx, sdkerrors.Wrap(sdkerrors.ErrInvalidRequest, "for eth tx AuthInfo Fee payer and granter should be empty")
 		}
 
-		if !authInfo.Fee.Amount.IsEqual(ethFeeAmount) {
-			return ctx, sdkerrors.Wrap(sdkerrors.ErrInvalidRequest, "invalid eth tx AuthInfo Fee Amount")
+		fmt.Println("fee compare", authInfo.Fee.Amount, txFee)
+		if !authInfo.Fee.Amount.IsEqual(txFee) {
```

Comment:
should we use `AmoutOfNoDenomValidation` instead

Reply:
it's to prevent tx malleability, so it's good to do a full comparison.
But it seems `IsEqual` will panic if denom don't match, is that a concern?

Reply:
let's avoid panic when there's no critical internal error